src/main/java/frc/robot/subsystems/ClimberSubsystem.java

```diff
@@ -30,19 +28,14 @@ public ClimberSubsystem() {
     m_motor = new SparkFlex(ClimberConstants.kMotorID, MotorType.kBrushless);
     SparkUtil.configureMotor(m_motor, ClimberConstants.kMotorConfig);
 
-    m_encoder = m_motor.getEncoder();
+    m_encoder = m_motor.getAbsoluteEncoder();
```

Comment:
The absolute encoder is not part of the motor (https://www.revrobotics.com/rev-11-1271/). Take a look at how we used a through-bore encoder for the gearbox last fall: https://github.com/TeraViks/arm2024/blob/main/src/main/java/frc/robot/subsystems/arm.java